Content.Shared/Doors/Components/DoorComponent.cs

```diff
@@ -133,20 +136,35 @@ public sealed partial class DoorComponent : Component
     /// </summary>
     [DataField, AutoNetworkedField]
     public HashSet<EntityUid> CurrentlyCrushing = new();
+
     #endregion
 
     #region Graphics
 
     /// <summary>
-    /// The key used when playing door opening/closing/emagging/deny animations.
+    /// The key used when playing door opening animations.
     /// </summary>
-    public const string AnimationKey = "door_animation";
+    public const string AnimationKeyOpen = "door_animation_open";
```

Comment:
Meaningful tech change one: how doors have been animated has been changed so there's four keys, and we cancel the key _other than the running one_ when doing an anim. So doors will rollback and do the correct anim if doing one that's wrong, but won't restart a running anim of the same type.